crates/sncast/src/helpers/scarb_utils.rs

```diff
@@ -389,7 +391,7 @@ mod tests {
         let metadata_err = get_scarb_metadata(&"Scarb.toml".into()).unwrap_err();
         assert!(metadata_err
             .to_string()
-            .contains("Failed to read Scarb.toml manifest file"));
+            .contains("Failed to read the `Scarb.toml` manifest file."));
```

Comment:
```suggestion
            .contains("Failed to read the `Scarb.toml` manifest file"));
```

We don't have periods in most other error messages

Reply:
mm, sure - replaced first one with `;` and removed the second one, to be more consistent